.github/workflows/pgindent.yml

```diff
@@ -0,0 +1,42 @@
+name: pgindent
+
+on:
+  push:
+    branches:
+      - main
+      - release
+    paths:
+      - 'pgxn/**.[ch]'
+      - 'pgxn/typdefs.list'
+  pull_request:
+    paths:
+      - 'pgxn/**.[ch]'
+      - 'pgxn/typdefs.list'
```

Comment:
I would also add a job to regenerate `pgxn/typdefs.list` (and probably `neon.typedefs`, but first we need to add it to the repo), and fail it if the diff is not empty. See an example in our autoscaling repo: https://github.com/neondatabase/autoscaling/blob/436edea2649b11bec85c078fa5fd65ddd718956c/.github/workflows/lint.yml#L38-L48

Reply:
> Actually pgxn/typedefs.list is obsolete, it was once manually added long time ago; my makefile rules generate it at the root of the repo. We should fix this.

Would you like me to add a commit to this PR which removes it?

Reply:
> Would you like me to add a commit to this PR which removes it?

Sure, go ahead.